src/test/java/com/prgrms/vouchermanager/service/CustomerServiceTest.java

```diff
@@ -0,0 +1,109 @@
+package com.prgrms.vouchermanager.service;
+
+import com.prgrms.vouchermanager.domain.customer.Customer;
+import com.prgrms.vouchermanager.exception.NotCorrectIdException;
+import com.prgrms.vouchermanager.repository.customer.CustomerRepository;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+import java.util.List;
+import java.util.UUID;
+
+@SpringBootTest
```

Comment:
```suggestion
@SpringBootTest
@ActiveProfiles("jdbc")
@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
```

서비스 레이어에서 데이터베이스를 사용하는 통합테스트로 보여집니다.
이때는 SpringBootTest가 필요하나 내장 DB를 사용하지 않으려면 @AutoConfigureTestDatabase 설정이 필요합니다.